cli/hzn.go

```diff
@@ -383,9 +383,9 @@ Environment Variables:
 	policyCmd := app.Command("policy", msgPrinter.Sprintf("List and manage policy for this Horizon edge node."))
 	policyListCmd := policyCmd.Command("list", msgPrinter.Sprintf("Display this edge node's policy."))
 	policyNewCmd := policyCmd.Command("new", msgPrinter.Sprintf("Display an empty policy template that can be filled in."))
-	policyUpdateCmd := policyCmd.Command("update", msgPrinter.Sprintf("Update the node's policy. The node's built-in properties will be automatically added if the input policy does not contain them."))
+	policyUpdateCmd := policyCmd.Command("update", msgPrinter.Sprintf("Create or replace the node's policy. The node's built-in properties will be automatically added if the input policy does not contain them."))
```

Comment:
Please change the message to:
Create or replace the node's policy. The node's built-in properties cannot be modified or deleted by this command, with the exception of openhorizon.allowPrivileged.

Reply:
done